README.md

```diff
@@ -91,16 +91,14 @@ To [use](https://autotools.io/pkgconfig/pkg_check_modules.html) with the GNU Aut
 To use with a CMake build environment, use the [FindPkgConfig](https://cmake.org/cmake/help/latest/module/FindPkgConfig.html) module. Here is an example:
 
 ```cmake
-    cmake_minimum_required(2.8 FATAL_ERROR)
+    cmake_minimum_required(3.4 FATAL_ERROR)
     project("MyPistacheProject")
-    
-    # Tell CMake to add support for pkg-config, then use it to find the library...
-    include(FindPkgConfig)
-    pkg_search_module(PISTACHE REQUIRED libpistache>=0.0.2)
-    
-    include_directories(${PISTACHE_INCLUDE_DIRS})
+
+    # Find the library.
+    find_package(Pistache 0.0.2 REQUIRED)
```

Comment:
As long as this doesn't break our `pkg-config` support, I'm fine with this.

Reply:
`pkg-config` has one change. It drops -latomic if system has atomic instruction support.
```
$ cat .build/libpistache.pc 
prefix=/usr/local
exec_prefix=/usr/local/bin
libdir=/usr/local/lib/
includedir=/usr/local/include/

Name: libpistache
URL: http://pistache.io/
Description: An elegant C++ REST framework.
Version: 0.0.002-git20200301
Requires:
Libs: -L'${libdir}/' -lpistache -lpthread -lssl -lcrypto
Libs.private:
Cflags: -I'${includedir}/'

```

Reply:
Got it. While you're at it, you might as well bump the git version as well since the above is for 1 March 2020.